composer.json

```diff
@@ -22,7 +22,9 @@
         "json-ld/tests": "@dev"
     },
     "autoload": {
-        "psr-0": { "ML\\JsonLD": "" }
+        "psr-4": { "ML\\JsonLD\\": "src/" }
```

Comment:
```json
"exclude-from-classmap": ["/Test/"]
```
should work.

See "Excluding paths from the optimized classmap" section of https://seld.be/notes/new-composer-patterns